egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -0,0 +1,190 @@
+#!/bin/bash
+
+# based on run_tdnn_6h.sh
+
+set -e
+
+# configs for 'chain'
+stage=12
+train_stage=-10
+get_egs_stage=-10
+dir=exp/chain/blstm_6h  # Note: _sp will get added to this if $speed_perturb == true.
+decode_iter=
+decode_dir_affix=
+
+# training options
+leftmost_questions_truncate=-1
+chunk_width=150
+chunk_left_context=40
+chunk_right_context=40
+xent_regularize=0.025
+
+label_delay=0
+# decode options
+extra_left_context=50
+extra_right_context=50
+frames_per_chunk=
+
+remove_egs=false
+common_egs_dir=
+
+affix=
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+# The iVector-extraction and feature-dumping parts are the same as the standard
+# nnet3 setup, and you can skip them by setting "--stage 8" if you have already
+# run those things.
+
+dir=$dir${affix:+_$affix}
+if [ $label_delay -gt 0 ]; then dir=${dir}_ld$label_delay; fi
+train_set=train_nodup_sp
+ali_dir=exp/tri5a_ali_nodup_sp
```

Comment:
The _sp alignments are generated by run_ivector_common.sh script called 
in other script. I can add a comment for it.

在 2016/4/14 0:52, Vijayaditya Peddinti 写道:
>
> In egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59583855>:
>
> > +if ! cuda-compiled; then
> > +  cat <<EOF && exit 1
> > +This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
> > +If you want to use GPUs (and have them), go to src/, and configure and make on a machine
> > +where "nvcc" is installed.
> > +EOF
> > +fi
> > +
> > +# The iVector-extraction and feature-dumping parts are the same as the standard
> > +# nnet3 setup, and you can skip them by setting "--stage 8" if you have already
> > +# run those things.
> > +
> > +dir=$dir${affix:+_$affix}
> > +if [ $label_delay -gt 0 ]; then dir=${dir}_ld$label_delay; fi
> > +train_set=train_nodup_sp
> > +ali_dir=exp/tri5a_ali_nodup_sp
>
> as you are not generating the alignments in the run_ivector_common.sh 
> script alignments will not be available for _sp directory.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59583855>
>